marlowe-playground-client/package.json

```diff
@@ -17,15 +17,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.2",
     "@popperjs/core": "^2.9.2",
-    "big-integer": "^1.6.50",
     "bignumber": "^1.1.0",
-    "blockly": "^4.20201217.0",
+    "blockly": "^9.3.3",
     "bootstrap": "^4.3.1",
     "decimal.js": "^10.0.0",
     "json-bigint": "^1.0.0",
-    "monaco-editor": "^0.22.3",
-    "monaco-emacs": "^0.2.2",
-    "monaco-vim": "^0.1.7",
+    "monaco-editor": "^0.38.0",
+    "monaco-emacs": "^0.3.0",
+    "monaco-vim": "^0.3.5",
```

Comment:
I think we should remove these in the future and the Editor mode selector... it is confusing, and something more in line with a "full IDE"... better to use `vim`, `emacs`, or `vscode` directly